docs/kinto-admin.rst

```diff
@@ -0,0 +1,260 @@
+Kinto Web Administration Console
+################################
+
+.. image:: images/kinto-admin-login.png
+    :align: center
+    :width: 800px
+
+
+
+A Web admin UI to manage data from a *Kinto* `<https://kinto.readthedocs.io/>`_ server: *Demo* `<http://kinto.github.io/kinto-admin/>`_.
+*kinto-admin* wants to be the *pgAdmin* `<http://pgadmin.org/>`_ for
+Kinto. You can also use it to build administration interfaces for
+Kinto-based systems.
+
+*Kinto-Admin* is used at Mozilla and released under the Apache v2 licence.
+
+
+.. _why-use:
+
+Why use Kinto-admin?
+====================
+
+- Expose your data over an HTTP interface, unlike databases like PostgreSQL
+- Use simple HTTP requests rather than SQL
+- Use `Kinto.js <https://kintojs.readthedocs.io/en/latest/>`_ to easily
+  implement offline first clients
+- Choose the database you want from those that Kinto supports, and use a
+  unified API to access its data
+- Manage your data using the handy
+  `admin UI <http://kinto.github.io/kinto-admin/>`_
+- Easily
+  :ref:`set up live push notifications <tutorial-notifications-websockets>`
+  for live updates of your application
+- Make it possible to
+  :ref:`share data between users <api-permissions>`
+  (using fine-grained permissions)
+- Take advantage of
+  :ref:`schema validation <collection-json-schema>`
+  if you need it
+- :ref:`Track changes <api-history>` on your application data
+
+
+.. _enable:
+
+Prerequisites
```

Comment:
I guess the next sections make more sense if we are developing Kinto Admin. If we are only using the plugin that is shipped with the server, which is already precompiled, I would simply give instructions on how to enable using a config file and a link to kinto admin repository if someone wants to customize or contribute.